fil-proofs-tooling/Cargo.toml

```diff
@@ -22,12 +22,12 @@ human-size = "0.4"
 prettytable-rs = "0.8"
 regex = "1.1.6"
 commandspec = "0.12.2"
-bellperson = ">=0.3.4"
 chrono = { version = "0.4.7", features = ["serde"] }
-fil-sapling-crypto = "0.1.2"
 memmap = "0.7.0"
-paired = "0.15.0"
-rand = "0.4"
+bellperson = { version = ">=0.3.4", git = "https://github.com/filecoin-project/bellman", branch = "update-from-upstream" }
```

Comment:
Remember to point `bellperson`, `fil-sapling-crypto`, and `paired` back to crates.io if/when new releases are cut.